### ISSUE TITLE
TypeError: substr() missing 1 required positional argument: 'x'

### ISSUE THREAD

**Issue by django-wong** [tomv564/LSP#129]
This happens no matter what I typed. No LSP completion show up

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 604, in on_query_completions
    res = vel.on_query_completions(prefix, locations)
  File "main in /Users/DjangoWong/Library/Application Support/Sublime Text 3/Installed Packages/LSP.sublime-package", line 1855, in on_query_completions
  File "main in /Users/DjangoWong/Library/Application Support/Sublime Text 3/Installed Packages/LSP.sublime-package", line 1880, in do_request
  File "main in /Users/DjangoWong/Library/Application Support/Sublime Text 3/Installed Packages/LSP.sublime-package", line 1843, in is_after_trigger_character
TypeError: substr() missing 1 required positional argument: 'x'
```

And my settings
```json
{
	"clients": {
		"phplsp": {
			"command": ["php", "/Users/DjangoWong/.composer/vendor/felixfbecker/language-server/bin/php-language-server.php"],
			"scopes": ["source.php"],
			"syntaxes": ["Packages/PHP/PHP.sublime-syntax"],
			"languageId": "php"
		}
	},
	"complete_all_chars": false,
	"only_show_lsp_completions": true,
	"resolve_completion_for_snippets": false
}
```

![image](https://user-images.githubusercontent.com/4019240/30873054-97619778-a2b1-11e7-8598-9e3c0906a034.png)

**Comment by django-wong** [comment]
If I set `complete_all_chars` to `true`, the error disappear but I can not see any other completions except LSP, even `only_show_lsp_completions` was `true`

for example, I want to see `serial_number` if I type `serial_`, but actually nothing.

![image](https://user-images.githubusercontent.com/4019240/30873778-f5582c64-a2b3-11e7-8287-55e9763ef4d0.png)

![image](https://user-images.githubusercontent.com/4019240/30873569-4baaa746-a2b3-11e7-869d-06586a835d28.png)

**Comment by tomv564** [comment]
Thanks for reporting, unfortunate regression. 

It looks like it is this commit accidentally remove `location - 1`: https://github.com/tomv564/LSP/commit/2306abe272bcc549ac9c67ef08f05d9af24bd448#diff-5bc02cefb3ea9e27f1a6776eabd1935dL1552

I just pushed a fix for this to master, if you have installed the package from the repository then you will get the update now - otherwise I will try to get a release out soon.

**Comment by django-wong** [comment]
Looking forward to the new release :)

**Comment by tomv564** [comment]
I released 0.2.2 yesterday to get this fix out, can you confirm it works for you now?

**Comment by django-wong** [comment]
Its works for me. Thanks
